packages/features/workspace_screen/lib/src/components/top_bar/overflow_menu.dart

```diff
@@ -132,6 +139,10 @@ class _OverflowMenuState extends ConsumerState<OverflowMenu> {
     return true;
   }
 
+  void _shareContent() {
+    Share.share('Check out this great app!');
```

Comment:
Do we leave 'Check out this great app!' or should we write something more specific?

Reply:
Something like the Catrobat website link? Or if you have something else in mind please let me know, and I will add it
Thank you